### PR TITLE
Update docs for v1.2.1 release

### LINKED ISSUE #36: Update docs for new tile release
AC TBA

Should include bullets / links to all relevant issues included in this release that require doc updates.

### PR DESCRIPTION
**Pull request by izgeri**
### What does this PR do?
Update docs for v1.2.1 release

### What ticket does this PR close?
Resolves #36 

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [x] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

### CODE REVIEW

File: docs-content/release-notes.html.md.erb [hunk 1]
```diff
@@ -34,7 +68,11 @@ Features in this release:
   could consume credentials for other services, if their field names exactly matched those used by 
   Conjur (e.g. version is a very common field).
 
-Known issues in this release: No known issues.
+Known issues in this release:
```
Comment (BradleyBoutcher) [inline docs-content/release-notes.html.md.erb:71]: Glad you added this retroactively!

File: docs-content/index.html.md.erb [hunk 2]
```diff
@@ -36,24 +36,24 @@ The following table provides version and version-support information about Cyber
     <th>Details</th>
     <tr>
         <td>Tile version</td>
-        <td>v1.2.0</td>
+        <td>v1.2.1</td>
     </tr>
     <tr>
         <td>Release date</td>
-        <td>August 7, 2020</td>
+        <td>January 14, 2021</td>
```
Comment (BradleyBoutcher) [inline docs-content/index.html.md.erb:43]: Technically I think the release date is yesterday (the 13th)

**Review by BradleyBoutcher — APPROVED** [review]
LGTM ✨

File: docs-content/release-notes.html.md.erb (hunk 1 above)
Comment (izgeri) [reply docs-content/release-notes.html.md.erb:71]: I actually just learned that this is recommended from here: https://github.com/pivotal-cf/docs-partners-template/blob/master/docs-content/release-notes.html.md.erb!!

**Pull request merged**